### PR TITLE
remove the global `/api/v2/job_events/` endpoint

### PR DESCRIPTION
**Pull request by ryanpetrello**

### CODE REVIEW

File: awx/api/views/__init__.py [hunk 1]
```diff
@@ -3766,6 +3759,13 @@ def get_serializer_context(self):
         context.update(no_truncate=True)
         return context
 
+    def get_queryset(self):
+        job = self.get_object().job
+        self.check_parent_access(job)
+        return super(JobEventDetail, self).get_queryset().filter(
+            job_created=job.created_or_epoch
```
Comment (ryanpetrello) [inline awx/api/views/__init__.py:3766]: `/api/v2/job_events/N/` should also add the proper `job_created` filter so the partition is selected properly

File: awx/api/views/__init__.py [hunk 2]
```diff
@@ -3779,7 +3779,11 @@ class JobEventChildrenList(NoTruncateMixin, SubListAPIView):
     def get_queryset(self):
         parent_event = self.get_parent_object()
         self.check_parent_access(parent_event)
-        qs = self.request.user.get_queryset(self.model).filter(parent_uuid=parent_event.uuid)
+        qs = self.request.user.get_queryset(self.model).filter(
+            parent_uuid=parent_event.uuid
+        ).filter(
+            job_created=parent_event.job.created_or_epoch
```
Comment (ryanpetrello) [inline awx/api/views/__init__.py:3785]: `/api/v2/job_events/N/children/` should also add the proper `job_created` filter so the partition is selected properly

**Review by jladdjr — APPROVED** [review]

File: awx/api/views/__init__.py (hunk 2 above)
Comment (jladdjr) [reply awx/api/views/__init__.py:3778]: great catch!

**Comment by jladdjr** [comment]
all looks good - merging!

**Pull request merged**